### ISSUE TITLE
Creating label bring blank page

### ISSUE THREAD

**Issue by ThierryBegin** [JordanKnott/taskcafe#121]
**Describe the bug**
When creating a label, the page get blank

**Expected behavior**
to have a window that allow me to create a label

**Screenshots / Live demo link**
![9LUhaW5k8R](https://user-images.githubusercontent.com/2735823/120699509-76069b80-c47e-11eb-962a-60dec2530ec3.gif)

**Additional context**
Docker install

**Comment by JackWolfard** [comment]
Porting description from #123 

**Describe the bug**

Given you're on a board page with no labels:
- Click on top right Labels drop down
- Press "Create a new label" button
- React crashes due to "current" being read-only 

**Expected behavior**

Prompt to add a new label to board

**Screenshots / Live demo link**

Console log detailing error:

<img width="479" alt="error_log" src="https://user-images.githubusercontent.com/31635014/122243405-194dac80-ce92-11eb-8a22-818485693ed6.png">

Source code snippet from [`frontend/src/Projects/Project/Board/index.tsx`](https://github.com/JordanKnott/taskcafe/blob/master/frontend/src/Projects/Project/Board/index.tsx#L476) line 476

```ts
...
if (data) {
    labelsRef.current = data.findProject.labels; // line producing error
    membersRef.current = data.findProject.members;
    ...
```

**Additional context**

- Running latest version of Taskcafé using Docker Compose in Firefox.
- Note: Line \#s are different from the reported console log and source code. I confirmed it was the same line manually.

**Comment by raslan** [comment]
Getting this same issue using Docker Compose on latest Chrome.

**Comment by JordanKnott** [comment]
I'm unable to replicate this using the `nightly` image - so I've released a new 'stable' image (0.3.4) that should fix this issue

**Comment by JordanKnott** [comment]
I was unable to replicate since it was only happening on production builds. I was able to replicate it and it is now fixed in version `0.3.6`
